.jenkins/docker/contrib/jenkins/configuration/scripts.groovy.d/on-gh-event.groovy

```diff
@@ -36,74 +36,37 @@ static Map exec(List args, File workingDirectory=null, Appendable stdout=null, A
                     println "Clone Url:${cloneUrl}"
                     println "Checkout Branch:${sourceBranch}"
 
-                    String selector = "env-id=${payload.number},env-name!=prod,github-owner=${repoOwner},github-repo=${repoName},!shared"
-                    // removed dev clean up:
-                    ['devops-sso-tools', 'devops-sso-sandbox'].each({ namespace ->
-                        //BuildConfig Output Images
-                        def ocGetBcRet = exec(['oc',"--namespace=${namespace}",'get','bc','-l',selector, '-o', 'jsonpath={range .items[*]}{.spec.output.to.namespace}/{.spec.output.to.name}{"\\n"}{end}'])
-                        println ocGetBcRet
-                        if (ocGetBcRet.status == 0 ){
-                            ocGetBcRet.out.toString().trim().split('\n').each({ item ->
-                                if (item.length() > 3){
-                                    def reference=item.split('/')
-                                    if (reference[0].length()==0){
-                                        reference[0]=namespace
-                                    }
-                                    println exec(['oc', "--namespace=${reference[0]}", 'tag', "${reference[1]}", '--delete=true'])
-                                }
-                            })
-                        }
-
-                        //DeploymentConfig Images
-                        def ocGetDcRet = exec(['oc',"--namespace=${namespace}",'get','dc','-l',selector, '-o', 'jsonpath={range .items[*]}{range .spec.triggers[*]}{.imageChangeParams.from.namespace}/{.imageChangeParams.from.name}{"\\n"}{end}{end}'])
-                        println ocGetDcRet
-                        if (ocGetDcRet.status == 0 ){
-                            ocGetDcRet.out.toString().trim().split('\n').each({ item ->
-                                if (item.length() > 3){
-                                    def reference=item.split('/')
-                                    if (reference[0].length()==0){
-                                        reference[0]=namespace
-                                    }
-                                    println exec(['oc', "--namespace=${reference[0]}", 'tag', "${reference[1]}", '--delete=true'])
-                                }
-                            })
-                        }
-
-                        //oc get dc -l 'env-id=pr-19,env-name!=prod' -o 'jsonpath={range .items[*]}{range .spec.triggers[*]}{.imageChangeParams.from.namespace}/{.imageChangeParams.from.name}{"\n"}{end}{end}'
-                        println exec(['oc', "--namespace=${namespace}", 'delete', 'all', '-l', selector])
-                        println exec(['oc', "--namespace=${namespace}", 'delete', 'PersistentVolumeClaim,Secret,ConfigMap,RoleBinding', '-l', selector])
-                    })
+                    println exec(['mkdir', '-p', gitWorkDir.getAbsolutePath()])
```

Comment:
how did you figuire this out?

Reply:
ref: https://github.com/bcgov/RSBC-DataHub-API